data_augmentation.py

```diff
@@ -0,0 +1,225 @@
+from skimage.transform import rescale
```

Comment:
This function needs to be moved to AxonDeepSeg/data_management/ so that it's being covered by the tests.